src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx

```diff
@@ -78,7 +80,35 @@ function RequestedAppointmentDetailsPage({
         ‹ <Link to="/requested">Manage appointments</Link>
       </div>
 
-      <h1>Pending {typeOfCareText} appointment</h1>
+      <h1>
+        {canceled ? 'Canceled' : 'Pending'} {typeOfCareText} appointment
+      </h1>
+      <AlertBox
+        status={canceled ? 'error' : 'info'}
+        className="vads-u-display--block vads-u-margin-bottom--2"
+        backgroundOnly
+      >
+        {canceled && 'This request has been canceled'}
+        {!canceled && (
+          <>
+            Your appointment request has been submitted. We will review your
+            request and contact you to schedule the first available appointment.
+            <div className="vads-u-display--flex vads-u-align-items--center vads-u-color--link-default vads-u-margin-top--2">
+              <i
+                aria-hidden="true"
+                className="fas fa-times vads-u-font-size--lg vads-u-font-weight--bold vads-u-margin-right--1"
+              />
+
+              <button
+                aria-label="Cancel request"
+                className="vaos-appts__cancel-btn va-button-link vads-u-flex--0"
+              >
+                Cancel Request
+              </button>
+            </div>
+          </>
+        )}
+      </AlertBox>
```

Comment:
I should have probably put this in a separate PR as it is out of scope of this ticket, but this small addition adds an `<AlertBox>` with the pending/canceled states for the request details.